<?xml version="1.0"?>
<launch>
  <arg name="prefix" default="wamv" />
  <node pkg="hector_mapping" type="hector_mapping" name="hector_mapping" output="screen">
    <param name="pub_map_odom_transform" value="true" />
    <param name="map_frame" value="map" />
    <param name="base_frame" value="$(arg prefix)/base_link"/>
    <param name="odom_frame" value="$(arg prefix)/odom" />
    <param name="map_resolution" value="0.5" />

    <!-- Map size/start point -->
    <param name="map_size" value="200" />
    <param name="map_start_x" value="0.5" />
    <param name="map_start_y" value="0.5" />
    <param name="map_multi_res_levels" value="2" />
    
    <!-- Map update parameters -->
    <param name="update_factor_free" value="0.4" />
    <param name="update_factor_occupied" value="0.9" />
    <param name="map_update_distance_thresh" value="0.4" />
    <param name="map_update_angle_thresh" value="0.06" />
    <param name="laser_z_min_value" value="-1.0" />
    <param name="laser_z_max_value" value="-1.0" />
  </node>
</launch>
